src/Pos/Wallet/Web/Api.hs

```diff
@@ -257,20 +256,13 @@ type UpdateTx =
     :> ReqBody '[JSON] CTxMeta
     :> WRes Post ()
 
-type GetHistory =
-       "txs"
-    :> "histories"
-    :> CCapture "walletId" CAccountId
-    :> QueryParam "skip" Word
-    :> QueryParam "limit" Word
-    :> WRes Get ([CTx], Word)
-
 type SearchHistory =
        "txs"
     :> "histories"
-    :> CCapture "walletId" CAccountId
-    :> Capture "search" Text
-    :> QueryParam "account" (CId Addr)
+    :> QueryParam "walletId" (CId Wal)
+    :> QueryParam "accountId" CAccountId
+    :> QueryParam "address" (CId Addr)
+    :> QueryParam "search" Text
```

Comment:
@flyingleafe `daedalus-bridge` builds successfully with this PR. Specifically, here is what I did to build it:

```
stack exec -- cardano-wallet-hs2purs
cd daedalus
npm install
npm run build:prod
```

There were no errors.